content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,282 @@ weight: 1
+    ✔ Ingress gateways installed
+    ✔ Ztunnel installed
+    ✔ Installation complete
+    {{< /text >}}
+
+## Deploy the sample application {#bookinfo}
+
+You’ll use the sample [bookinfo application](/docs/examples/bookinfo/), which is part of
+the Istio distribution that you downloaded above. In ambient mode, you deploy applications to
+your Kubernetes cluster exactly the same way you would
+without Istio. This means that you can have your applications running in your cluster before
+you enable ambient mesh and have them join the mesh without needing to restart or
+reconfigure them.
+
+{{< warning >}}
+Make sure the default namespace does not include the label `istio-injection=enabled` because when using ambient you do not want Istio to inject sidecars into the application pods.
+{{< /warning >}}
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -f samples/sleep/sleep.yaml
+$ kubectl apply -f samples/sleep/notsleep.yaml
+{{< /text >}}
+
+Note: `sleep` and `notsleep` are two simple applications that can serve as curl clients.
+
+Connect `productpage` to the Istio ingress gateway so you can access the bookinfo
+app from outside of the cluster:
+
+{{< text bash >}}
+$ kubectl apply -f samples/bookinfo/networking/bookinfo-gateway.yaml
+{{< /text >}}
+
+Test your bookinfo application, it should work with or without the gateway. Note: you can replace `istio-ingressgateway.istio-system` below with its load balancer IP (or hostname) if it has one:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | head -n1
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | head -n1
+$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | head -n1
+{{< /text >}}
+
+## Adding your application to ambient {#addtoambient}
+
+You can enable all pods in a given namespace to be part of the ambient mesh
+by simply labeling the namespace:
+
+{{< text bash >}}
+$ kubectl label namespace default istio.io/dataplane-mode=ambient
+{{< /text >}}
+
+Congratulations! You have successfully added all pods in the default namespace
+to the ambient mesh. The best part is that there was no need to restart or redeploy anything!
+
+Send some test traffic:
+
+{{< text bash >}}
+$ kubectl exec deploy/sleep -- curl -s http://istio-ingressgateway.istio-system/productpage | head -n10
+$ kubectl exec deploy/sleep -- curl -s http://productpage:9080/ | head -n10
+$ kubectl exec deploy/notsleep -- curl -s http://productpage:9080/ | head -n10
+{{< /text >}}
+
+You’ll immediately gain mTLS communication and L4 telemetry among the applications in the Ambient mesh.
+If you follow the instructions to install [Prometheus](/docs/ops/integrations/prometheus/#installation)
+and [Kiali](/docs/ops/integrations/kiali/#installation), you’ll be able to visualize your application
+in Kiali’s dashboard:
+
+{{< image link="./kiali-ambient.png" caption="Kiali Dashboard" >}}
```

Comment:
This is not the best diagram to use here, since it shows the helloworld service, while the instructions are using bookinfo :(

Reply:
Oh, my bad, i put in the wrong pic :) will correct :)